xonsh/completers/python.py

```diff
@@ -273,7 +273,7 @@ def python_signature_complete(prefix, line, end, ctx, filter_func):
         return set()
     try:
         sig = inspect.signature(val)
-    except ValueError:
+    except (ValueError, TypeError):
```

Comment:
I am not sure this is the correct way. For deprecation warning, one should [filter](https://stackoverflow.com/questions/879173/how-to-ignore-deprecation-warnings-in-python) it in their config
(.xonshrc)

Reply:
It's a bug in the autocomplete that will spam the console mid-typing